_schemas/1.0/tests/request/resource/create/invalid/relationship_with_bad_resource_identifier.json

```
@@ -0,0 +1,15 @@
+{
+  "data": {
+    "type": "article",
+    "attributes": {
+      "title": "PHP is so crazy !"
```

Do you mind changing to a more neutral title in the tests? This might be offending to some users of JSON:API specification.

```suggestion
      "title": "Resource identifier object used for relationship data is wrong."
```

Same applies to the other cases as well.

---

I agree with you. I think this is a copy/paste of an example found in an older version of the specs.

---

Done